kuma/wiki/tests/test_templates.py

```diff
@@ -680,6 +680,7 @@ def test_tag_list(self):
         doc = pq(response.content)
         eq_(1, len(doc('#document-list ul.document-list li')))
 
+    @pytest.mark.xfail(reason='Requires migrations')
```

Comment:
I'm confused as to how we're just now noticing this problem. Shouldn't this test be failing when run against the DBs in SCL3 where the custom collation was applied?